servers/src/grin/sync/header_sync.rs

```diff
@@ -142,7 +142,7 @@ impl HeaderSync {
 						match self.sync_state.status() {
 							SyncStatus::HeaderSync { .. } | SyncStatus::BodySync { .. } => {
 								// Ban this fraud peer which claims a higher work but can't send us the real headers
-								if now > *stalling_ts + Duration::seconds(120)
+								if now > *stalling_ts + Duration::seconds(20)
```

Comment:
I need to check the code in details, but I saw today that:
* we re-request headers after 10 seconds
* we don't ban a peers which doesn't reply to get headers request
so probably this code doesn't work as expected

Reply:
My understanding of it is that if the peer is not sending the requested headers we will keep asking every 10 seconds up till 120 seconds at which point they are banned.